codelabs/grpc-java-opentelemetry/README.md

```diff
@@ -15,4 +15,384 @@ Designed for developers already familiar with gRPC and wanting to learn how to i
 - [start_here](start_here/) directory serves as a starting point for the
   codelab.
 - [complete](complete/) directory showcases the finished code, giving you a
-  peek of how the final implementation should look like.
\ No newline at end of file
+  peek of how the final implementation should look like.
+
+## Before you Begin
+
+### What you’ll learn
+
+* How to setup OpenTelemetry for existing gRPC Java application
+* Running a local Prometheus instance
+* Exporting metrics to Prometheus
+* View metrics from Prometheus dashboard
+
+### What you’ll need
+
+* A computer with internet connection
+
+### Prerequisites
+
+* [JDK](https://jdk.java.net/) version 8 or higher
+* Use [the current directory](https://github.com/grpc-ecosystem/grpc-codelabs/tree/main/codelabs/grpc-java-opentelemetry) as a starting point for this codelab.
```

Comment:
I think we should point to the `start_here` dir.  Or link the download link like: 

https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2Fgrpc-ecosystem%2Fgrpc-codelabs%2Ftree%2Fmain%2Fcodelabs%2Fgrpc-java-opentelemetry%2Fstart_here

Reply:
Instead of copying gradle and its required files to both repositories, I've added it in the directory common to both `start_here` and `complete`. So downloading only `start_here` will not be sufficient.